docs/diagnostics/PX1022.md

```diff
@@ -5,13 +5,15 @@ This document describes the PX1022 diagnostic.
 
 | Code   | Short Description                                                                                                | Type  | Code Fix    | 
 | ------ | ---------------------------------------------------------------------------------------------------------------- | ----- | ----------- | 
-| PX1022 | The non-public graph extensions are not supported.                                                               | Error | Available   |
+| PX1022 | The non-public graph and DAC extensions are not supported.                                                       | Error | Available   |
 
 ## Diagnostic Description
-You can declare a class that is a graph extension. The class must be public.
+You can declare a class that is a graph extension. The Acumatica Customization Platform recognizes only public graph or DAC extensions.
 
 To fix the issue, you can make the graph extension public.
 
+The code fix adds the `public` key word to the class declaration.
```

Comment:
It replaces the current accessibility modifier with a public for the class declaration and also, in case of nested types, for all containing non-public type declarations.